spec/requests/new_document_spec.rb

```diff
@@ -1,64 +1,58 @@
 RSpec.describe "New Document" do
-  describe "GET /documents/choose-document-type" do
-    it "shows supertype when a supertype managed by content publisher is selected" do
-      supertype = Supertype.all.reject(&:managed_elsewhere).first
-      get choose_document_type_path, params: { supertype: supertype.id }
+  describe "GET /documents/show" do
+    it "shows the root document type selection when no selection has been made" do
+      get new_document_path
       expect(response).to have_http_status(:ok)
-      expect(response.body).to have_content(I18n.t!("supertypes.#{supertype.id}.label"))
+      expect(response.body).to have_content(I18n.t("document_type_selections.root.label"))
     end
 
-    it "redirects when a supertype managed elsewhere is selected" do
-      supertype = Supertype.all.find(&:managed_elsewhere)
-      get choose_document_type_path, params: { supertype: supertype.id }
-      expect(response).to redirect_to(supertype.managed_elsewhere_url)
-    end
-
-    it "redirects to guidance page when \"I'm not sure\" option is selected" do
-      supertype = Supertype.all.find { |type| type.id == "not-sure" }
-      get choose_document_type_path, params: { supertype: supertype.id }
-      expect(response).to redirect_to(guidance_url)
+    it "shows the page for the selected document type" do
+      get new_document_path, params: { type: "news" }
+      expect(response).to have_http_status(:ok)
+      expect(response.body).to have_content("News story")
```

Comment:
I think this text is defined in an i18n file so consistency would dictate we assert based on the I18n key rather than the text directly.